### DISCUSSION TITLE
Native Apple Silicon support

### DISCUSSION THREAD

**Discussion by pmcarrion** [Lennolium/swiftGuard#8] [Announcements]
Please add native support for Apple Silicon in swiftGuard.
Thanks.

**Comment by Lennolium** [comment]
Thanks for your feature request. 

As you can see in the [Roadmap](https://github.com/Lennolium/swiftGuard#roadmap), I am currently working on native M1/M2 support. Some dependencies are buggy on Arm, thats why I need to refactor before packaging for apple silicon :)

Until next release you can still use swiftGuard with apple rosetta, even if the performance is a little bit slower :)

**Comment by Bubba8291** [comment]
@pmcarrion @Lennolium #19

**Comment by Lennolium** [comment]
Native arm64 version is downloadable from the release tab.

**Reply by snw7** [reply]
It still says last release in 2023. (I can download the new version though.) Still I believe it would be less confusing, if you created a new release with the current date :)

**Reply by Lennolium** [reply]
Yes, I know. That was not good practice from my side. I feel so bad to say it again, but the big release is almost finished. I'm working really hard on it and I do not want to deliver a imperfect app. Sorry for the delay 😭

**Reply by snw7** [reply]
Dont worry :) Looking forward to it!
